code/game/turfs/turf.dm

```diff
@@ -169,6 +171,8 @@
 		air_master.remove_from_active(src)
 
 	var/turf/W = new path(src)
+	if(!defer_change)
+		W.AfterChange()
 
 	if(istype(W, /turf/simulated))
 		W:Assimilate_Air()
```

Comment:
I do know what should be done.
I've traced down the root of the problem to `update_icon` of the atmos pipes - <a href=https://github.com/ParadiseSS13/Paradise/blob/master/code/ATMOSPHERICS/pipes/simple/pipe_simple.dm#L144-L151>if called without the nodes set up, the whole system comes crashing down.</a>

I am currently working on getting the loader not to call `update_icon`, although I have the idea this may not be the best solution to such a problem. Nonetheless, I do not want to add a `process` that would bog down the server for such a common machinery object, so advice on how to prevent free-floating pipes would be appreciated. Perhaps just watching for free-floating checks only on pipe wrenching and disconnection... We'll see.